### DISCUSSION TITLE
Understanding project concept

### DISCUSSION THREAD

**Discussion by kinsleykajiva** [Charles92011/webrtc-java-experimental-server#4] [General]
Hi so I have been looking at the read me of the project, I just wish to further understand a bit what is the project really trying to achieve may be the use case may be if that can help as well.?

**Comment by Charles92011** [comment]
@kinsleykajiva the goal of this project is to learn how to use the JavaRTC library in a purpose-built server. There's no other purpose than that.

**Reply by kinsleykajiva** [reply]
so is there a limit to the type of transport for the clients connecting to the server?

**Reply by Charles92011** [reply]
I didn't set a limit in the project. 
It's experimental and I didn't want to add anything to complicate the project. A limitation is easy to add if it's desired, but I didn't want to add anything unnecessary to the project.
